js/test/durfmt/testdurfmt.js

```diff
@@ -4943,7 +4943,7 @@ module.exports.testdurfmt = {
             second: 1,
             millisecond: 1
         });
-        test.equal(duration.toString(), "1년 1개월 1주 1일 1시간 1분 1초 1밀리초");
+        test.equal(duration.toString(), "1년 1개월 1주 1일 1시간 1분 1초 1ms");
```

Comment:
Actually, I've been manually fixed this part to Korean, even though the CLDR provides the `ms`.

Reply:
Then I will introduce code into the cldr tool to override this then.